issue4/navigation_test.js

```diff
@@ -0,0 +1,61 @@
+var file_path = process.env.FILE_PATH
+var webdriver = require('selenium-webdriver');
+var assert = require('chai').assert;
+var { Builder, By } = require("selenium-webdriver");
+
+
+
+describe("Testing Navigation on HtmlForms", function() {
+    this.timeout(3 * 1000 * 60);
+    var driver = new webdriver.Builder().forBrowser('chrome').build()
+
+    before(function setupWebdriver(done) {
+        var file_url = `file:///Users/user/Downloads/Dufuna-CodeCamp20/submissions/oyetejuOloyede/htmlForms/contact.html`;
+       
+        driver.get(file_url).then(done)
+    })
+
+        after(function() {
+            driver.quit()
+        })
+
+    it("test case: check if Contact Us link is present",  async () => {
+        const element = await driver.findElement(By.css("a[href='../htmlForms/contact.html']"));
```

Comment:
Are we sure that the file will always be in a folder named `htmlForms` and will the file itself always be named `contact.html`?